students/biegon_piotr/lesson_01_basics/Sum_of_digits.py

```diff
@@ -0,0 +1,5 @@
+print("Sum of digits\n")
+
+number = int(input("Enter a three-digit number: "))
+
+print("\nSum of a three-digit number is:", (sum([int(digit) for digit in str(number)])))
```

Comment:
unnecessary parenthesis.

It's okay, but in the real world, we should avoid casting to string and then back to int.